docroot/modules/contrib/mirador/mirador_annotation/mirador_annotation.info.yml

```diff
@@ -11,3 +11,4 @@ core: '8.x'
 dependencies:
   - rest
   - mirador
+  - hal
```

Comment:
Hmmm.. Ideally we should be able to achieve what we trying only by using JSON format. Will check annotation store & update you.